Python/Twitter_Hashtags/README.md

```diff
@@ -0,0 +1,37 @@
+# Tweet scraper using a particular hashtag without the intervention of twitter API
+
+- Here, we make use of snscrape to scrape tweets associated with a particular hashtag. Snscrape is a python library that scrapes twitter without the use of API keys.
+
+- In this script, we take the input of a hashtag and generate details associated with it (Eg., id, username, date, time, the tweet content, number of replies, likes, retweets, quotes and also its URL) tracing upto the desired number of tweets. 
+
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscript library: (Supported for Python 3.8 and above only) It's development version is installed as
```

Comment:
Is it beneficial for us? Many Users (including me) use either Python 3.6/3.7.

Reply:
Snscrape works in python version 3.6 as well, however we cannot have access to replyCount, likeCount and so on, which is available in the development version. For python 3.6, we can get id, url, the actual tweet and username? Should I make changes accordingly then for 3.6 and pypi snscrape?

Reply:
Its understandable. We can hack our way around this.